columnflow/plotting/plot_util.py

```diff
@@ -172,7 +174,8 @@ def prepare_style_config(
 
     # disable minor ticks based on variable_inst
     if variable_inst.discrete_x:
-        style_config["ax_cfg"]["xticks"] = range(int(xlim[0]), int(xlim[1]) + 1)
```

Comment:
I had a variable with binning `(100, 0.0, 1.0e10)` which caused `range` to create 10B integers in memory. Changed this to `bin_edges` for now, but this isn't optimal either.

Reply:
Resolving this for now, should be followed up on in #207.